server/tests/roadmaps.test.ts

```diff
@@ -106,9 +179,11 @@ describe('Test /roadmaps/ api', function () {
         Permission.RoadmapDelete,
         () => loggedInAgent.delete('/roadmaps/' + firstRoadmapId),
       );
+      const firstRoadmapAfterDelete = (await Roadmap.query().first()).id;
```

Comment:
The order of the roadmaps can not be depended on, so better would be to check if roadmap with the same id still exists.